lnwire/funding_locked.go

```diff
@@ -0,0 +1,151 @@
+package lnwire
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/roasbeef/btcd/btcec"
+	"github.com/roasbeef/btcd/wire"
+)
+
+type FundingLocked struct {
+	ChannelOutpoint *wire.OutPoint
```

Comment:
Good catch. I did this so that we don't have to persist the temporary channel ids to the database. We may potentially revise the spec for funding_locked.